recipes/faunus/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "faunus" %}
+{% set version = "2.1.1" %}
+{% set commit = "afba113c6008139fab704e682305847b5ae1d872" %}
+{% set sha256 = "8a224f8be892b06026de21e85d5c42ab30f81b9044917bdc87afcc81834c7044" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # uncommented while staged
+  # fn: {{ version }}.tar.gz
+  # url: https://github.com/mlund/{{ name }}/archive/v{{ version }}.tar.gz
```

Comment:
Did you need to deal with this?

Reply:
EG remove the comments?

Reply:
OR remove the lines below it?

Reply:
I left it like this to accommodate potential changes to the build system specific to conda-forge. If you think the recipe is ready to go, I'll release v2.1.1 and target that.

Reply:
tuned in on v2.1.1; comments removed